### ISSUE TITLE
sync.connect hangs

### ISSUE THREAD

**Issue by evanshortiss** [feedhenry/fh-sync#34]
Just a heads up. In release 1.0.5 there's a clause like so in `mongodbQueue.js` that causes sync startup to fail due to a missing `else`:

```js
if (!existingIndex) {
  return callback(null, false);
} else if (existingIndex.expireAfterSeconds !== self.queueOptions.ttl) {
  return callback(null, true);
}
```

Not sure what the `else` should be here so flagging for input @wtrocki. The `master` branch does not have this so I assume it's on some other branch?

**Comment by evanshortiss** [comment]
I worked around it by deleting the sync queue and other collections in MongoDB for now.

**Comment by wtrocki** [comment]
See: https://github.com/feedhenry/fh-sync/pull/29/files

**Comment by wtrocki** [comment]
@evanshortiss Thanks for reporting bug. I see this change on master. 1.0.5 was broken. It was fixed in 1.0.7, but maybe there is another case I'm missing.

**Comment by evanshortiss** [comment]
@wtrocki nice, it was just the specific release I had so. Looks good to close this!
